stages/org.osbuild.grub2

```diff
@@ -222,6 +222,14 @@ class GrubConfig:
             "features": features,
         }
 
+        if remove_unset:
+            # set all remaining config options to empty strings in case they are not set
```

Comment:
I understand but there's no condition here if the keys are actually unset; this overwrites the keys if they are already present maybe the comment should be worded differently?

```
>>> D = {"a": "b"}
>>> D.update({"a": "c"})
>>> D
{'a': 'c'}
>>> D = {"a": "b"}
>>> D.update({"a": "c"})
>>> D
{'a': 'c'}
```